src/css.js

```diff
@@ -28,6 +28,7 @@ var
 	// except "table", "table-cell", or "table-caption"
 	// See here for display values: https://developer.mozilla.org/en-US/docs/CSS/display
 	rdisplayswap = /^(none|table(?!-c[ea]).+)/,
+	rnonemptywhite = /^\s+$/,
```

Comment:
> it collapses other whitespaces as well

Jeez, in that case it's doubly broken. :\ I really hope we can pull out this change quickly.